### PR TITLE
Text Translator

### PR DESCRIPTION
**Pull request by chrisdedman**
Pull Request related to this issue https://github.com/Code-Society-Lab/grace/issues/16

### CODE REVIEW

File: bot/extensions/translator_cog.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from discord.ext.commands import Cog, hybrid_command
+from googletrans import Translator
+import googletrans
```
Comment (PenguinBoi12) [inline bot/extensions/translator_cog.py:3]: Unused import goes...

Comment (PenguinBoi12) [inline bot/extensions/translator_cog.py:3]: Unused import goes...

File: bot/extensions/translator_cog.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+from discord import Embed
+
+
+class TranslatorCog(Cog, name="Translator", description="Translate sentances from any languages to English."):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @hybrid_command(name='translator', help='Translate a sentence/word from any languages to English', \
```
Comment (PenguinBoi12) [inline bot/extensions/translator_cog.py:11]: This is how it should be formatted (see bellow). The `\` is  unnecessary and the usage should go right under the name. 

```python
    @hybrid_command(name='translator', help='Translate a sentence/word from any languages to English',
                    usage="sentence={sentence}")
```

File: setup.py [hunk 3]
```diff
@@ -42,7 +42,8 @@ def after_install():
         'mypy',
         'alembic',
         'configparser',
-        'pygithub'
+        'pygithub',
+        'googletrans==4.0.0-rc1'
```
Comment (PenguinBoi12) [inline setup.py:46]: Is there a really specific reason behind this specific version?

**Review by PenguinBoi12 — APPROVED** [review]

**Review by PenguinBoi12 — APPROVED** [review]
I love that feature. Appart from really minor formatting issue all is good. I also left a question about the library version but as soon as I get an adequate answer it's good to merge :slightly_smiling_face:.

File: setup.py (hunk 3 above)
Comment (chrisdedman) [reply setup.py:46]: Yes, if I don't specify this version, their is a bug (newest version from what I found on google). That's why i need this specific version.

Comment (PenguinBoi12) [reply setup.py:46]: Okay perfect then!

File: bot/extensions/translator_cog.py (hunk 2 above)
Comment (chrisdedman) [reply bot/extensions/translator_cog.py]: Oh, yes. My IDE was messing with me when I did like you did for some reason lol. Thank you

**Pull request merged**